src/routes/safe-apps/entities/safe-app.entity.ts

```diff
@@ -0,0 +1,25 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { SafeApp as DomainSafeApp } from '../../../domain/safe-apps/entities/safe-app.entity';
+import { SafeAppAccessControl } from './safe-app-access-control.entity';
+import { SafeAppProvider } from './safe-app-provider.entity';
+
+export class SafeApp implements DomainSafeApp {
+  @ApiProperty()
+  id: number;
+  @ApiProperty()
+  url: string;
+  @ApiProperty()
+  name: string;
+  @ApiProperty()
+  iconUrl: string;
+  @ApiProperty()
+  description: string;
+  @ApiProperty()
+  chainIds: string[];
+  @ApiProperty()
```

Comment:
Optional `ApiProperty`

Reply:
Changed in [21ff50f](https://github.com/5afe/safe-client-gateway-nest/commit/21ff50f7bc77e3cb11bcbaa2c1cf0322c60cd08c)